src/tests/clipperUI/components/previewViewer/augmentationPreview_tests.tsx

```diff
@@ -29,13 +25,64 @@ export class AugmentationPreviewTests extends TestModule {
 	}
 
 	protected tests() {
+		function isPositive(value: number): number {
+			if (value > 0) {
+				return 1;
+			} else {
+				return 0;
+			}
+		}
 		test("The tab order flow from the header to the preview title is correct in Augmentation mode", () => {
 			let mockClipperState = this.getMockAugmentationModeState();
 			let defaultComponent = <AugmentationPreview clipperState={mockClipperState} />;
 			MithrilUtils.mountToFixture(defaultComponent);
 
-			Assert.tabOrderIsIncremental([Constants.Ids.highlightButton, Constants.Ids.sansSerif, Constants.Ids.serif, Constants.Ids.decrementFontSize,
+			MithrilUtils.simulateAction(() => {
+				document.getElementById(Constants.Ids.serif).click();
+			});
+			Assert.tabOrderIsIncremental([Constants.Ids.highlightButton, Constants.Ids.serif, Constants.Ids.decrementFontSize,
 				Constants.Ids.incrementFontSize, Constants.Ids.previewHeaderInput]);
+
+			MithrilUtils.simulateAction(() => {
+				document.getElementById(Constants.Ids.sansSerif).click();
+			});
+			Assert.tabOrderIsIncremental([Constants.Ids.highlightButton, Constants.Ids.sansSerif, Constants.Ids.decrementFontSize,
+				Constants.Ids.incrementFontSize, Constants.Ids.previewHeaderInput]);
+		});
+
+		test("When a pair of codependent buttons have conditional attributes, they both respond with the same behavior when clicked and have different values from each other", () => {
+			let mockClipperState = this.getMockAugmentationModeState();
+			let defaultComponent = <AugmentationPreview clipperState={mockClipperState} />;
+			MithrilUtils.mountToFixture(defaultComponent);
+			let unSelectedTabShouldBeNegative = -1;
```

Comment:
No need to declare a variable for this. It's easier to read strictEqual(unselectedTabIndices[0], -1, ...